Automattic/wordpress-importer
# Try building with `@wordpress/scripts`

This looks to build our JS file with webpack, configured with the `@wordpress/webpack-config` config file.

### Notes

- I saw issues with the `source-map-loader` package not being found, so installed it directly for now. I'm not sure whether we should keep this or fix it some other way.
- I'm marking this PR as blocked until the dust settles and the `@wordpress/scripts` related PR (https://github.com/WordPress/gutenberg/pull/13814) gets landed :)

### Testing

- Make sure you have gutenberg updated to branch `pr/12685`
- Run `npm run build` in gutenberg's root directory
- `cd` in to `gutenberg/packages/scripts`
- Run `npm link` here (not in gutenberg itself)
- Checkout this branch in `wordpress-importer`
- From the `wordpress-importer`'s root directory, run `npm link @wordpress/scripts`
- Run `npm install`
- Run `npm run build` in `wordpress-importer`'s root directory
- Open your test jetpack at `/wp-admin/admin.php?import=wordpress`
- You should see the `Drop something here` screen. 
- You can try editing the javascript and re-running the build script from `wordpress-importer` to make sure those changes are reflected
